internal/flag/flags.go

```diff
@@ -283,6 +283,8 @@ const (
 	DisableSchemaAdvisor                      = "disableSchemaAdvisor"                      // DisableSchemaAdvisor flag
 	EnableRealtimePerformancePanel            = "enableRealtimePerformancePanel"            // EnableRealtimePerformancePanel flag
 	DisableRealtimePerformancePanel           = "disableRealtimePerformancePanel"           // DisableRealtimePerformancePanel flag
+	CloudProvider                             = "cloudProvider"                             // CloudProvider flag
+	IamRoleID                                 = "iamRoleID"                                 // IamRoleID flag
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
	IAMRoleID                                 = "iamRoleId"                                 // IamRoleID flag
```